opencti-platform/opencti-front/lang/front/fr.json

```
@@ -1206,6 +1210,8 @@
   "Enrichment connector": "Connecteur d'enrichissement",
   "Enrichment connectors": "Connecteurs d'enrichissement",
   "Enroll in playbook": "Envoyer vers un playbook",
+  "enrollment_confirmation_dialog": "'La procédure d'inscription est toujours en cours. La fermeture de cette boîte de dialogue mettra fin à l'inscription. Êtes-vous sûr de vouloir fermer ?",
```

Just a little `'` here
```suggestion
  "enrollment_confirmation_dialog": "La procédure d'inscription est toujours en cours. La fermeture de cette boîte de dialogue mettra fin à l'inscription. Êtes-vous sûr de vouloir fermer ?",
```